demo/ThemeTransform.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang=en>
+
+  <head>
+    <meta charset=utf-8>
+    <title>ThemeTransform</title>
+    <style>
+      :root {
+        --control-panel-theme-visibility: visible;
+      }
+      body{
+        padding: 20px 20px 20px 300px !important;
+      }
+      .content_block {
+        padding-bottom: 50px;
+      }
+      #demoMenu a {
+        display: block;
+        color: black;
+      }
+      #demoTools {
+        background: #ccc;
+        padding: 15px;
+        position: fixed;
+        left: 20px;
+        top: 20px;
+        z-index: 10000;
+        width: 230px;
+        border-radius: 8px;
+      }
+      .demoArticleTitle {
+        color: red;
+        font-size: 2.0em;
+      }
+    </style>
+ 
+    <link href="https://en.wikipedia.org/w/load.php?debug=false&lang=en&only=styles&skin=vector&modules=skins.minerva.base.reset|skins.minerva.content.styles|ext.math.styles|" rel="stylesheet" type="text/css"></link>
+
+    <script src=build/wikimedia-page-library-transform.js></script>
+    <link rel=import href=ControlPanel.html>
+  </head>
+
+  <body>
+    <div id='demoTools'>
+      <control-panel></control-panel>
+      <div id='demoMenu'></div>
+    </div>
+    <div class='content' id='content'></div>
+    
+    <script>
+        
+(function() {
+
+  // The json files below were saved using the mobileview parameters in the URL below.
+  // eslint-disable-next-line max-len
```

Comment:
Please just disable the rule for URLs in the ESLint config. I've found the following works well: max-len: `[1, {tabWidth: 2, ignoreUrls: true, ignorePattern: '^/// .+'}]`.